cmd/contour/serve.go

```diff
@@ -419,6 +424,12 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	// the output of the other processors.
 	dagProcessors = append(dagProcessors, &dag.ListenerProcessor{})
 
+	// Setup a Manager
+	mgr, err := manager.New(controller_config.GetConfigOrDie(), manager.Options{})
+	if err != nil {
+		log.Fatal(err, "unable to set up overall controller manager")
```

Comment:
I suggest dropping"overall", maybe something like:
"unable to create contour manager" or "unable to create contour server manager".

Reply:
I dropped "overall".